src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -35,4 +35,4 @@
 
     @NotNull
     private int price;
-}
+}
```

Comment:
?

Reply:
Pobrałem zmiany z maina, więc to jest powód.

Reply:
To jest niemożliwe, GitHub porównuje Twój branch z branchem remote main i to są różnice. Musisz mieć nieaktualny main

Reply:
U mnie - mam aktualny main - oba te pliki mają nową linię na końcu pliku. W tych zmianach ta nowa linia jest usuwana